### PR TITLE
Expand OA definition

### LINKED ISSUE #263: [PE-D][Tester A] Glossary terms in UG
Maybe can explain that OA stands for Online Assessment as all users may not know what it means

Screenshot 2024-11-08 at 6.33.00 PM

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: Raghava-Chittidi/ped#21

### PR DESCRIPTION
**Pull request by Shux347**
closes #263

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -508,7 +508,7 @@ _Details coming soon ..._
 - **JSON** (JavaScript Object Notation): A lightweight data-interchange format that is easy for humans to read and write and for machines to parse and generate. InternBuddy uses JSON to store its data files.
 - **Index**: A number used to identify the position of a company or application in a list. InternBuddy commands often require an index to reference a specific company or application.
 - **Home Folder**: The directory where InternBuddy stores its data and related files on your computer.
-- **APPLIED / OA / INTERVIEWED / OFFERED / ACCEPTED / REJECTED**: The various statuses that can describe an application’s progress in the hiring process within InternBuddy.
+- **APPLIED / OA (Online Assessment) / INTERVIEWED / OFFERED / ACCEPTED / REJECTED**: The various statuses that can describe an application’s progress in the hiring process within InternBuddy.
```
Comment (blanklogic) [inline docs/UserGuide.md:511]: Simple fix to the User Guide with better clarity.

**Review by blanklogic — APPROVED** [review]
LGTM!

**Review by uniqly — APPROVED** [review]
LGTM!

**Review by realqijun — APPROVED** [review]
LGTM!

**Pull request merged**